libs/platform/src/lib/components/link/link.component.ts

```diff
@@ -1,21 +1,3 @@
-/**
```

Comment:
this should be addressed in a separate PR

Reply:
keeping only link component changes. removed formatting changes from this PR.